tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "baseUrl": "src",
+    "declaration": true,
+    "incremental": true,
+    // "lib": ["dom"],
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "noEmit": true,
+    "noUnusedLocals": true,
+    "outDir": "dist",
+    "pretty": true,
+    "rootDir": "src",
+    "skipLibCheck": true,
+    "sourceMap": true,
+    // "strict": false,
```

Comment:
Its highly advised to start with strict mode turned on when adding types to get maximal safety (and save a lot of headaches down the road). Converting a sizable project with this turned on from the get go, however, is way more difficult; you might want to keep this set to false and move things incrementally.